sdk/manifests/apiextensions.k8s.io_v1_customresourcedefinition_releases.release.giantswarm.io.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Release CRD built by kustomize. It takes Release CRD from `config/crd/bases/release.giantswarm.io_releases.yaml` and applies replacement for updating CR name pattern.